splunk_eventgen/lib/plugins/generator/replay.py

```diff
@@ -173,9 +173,12 @@ def gen(self, count, earliest, latest, samplename=None):
                     rpevent, self.backfill_time, earliest, latest
                 )
                 previous_event = current_event
-                previous_event_timediff = rpevent["timediff"]
+                previous_event_timediff = line_list[index + 1]["timediff"]
```

Comment:
If you have a jsonl file with one line, you would run into a problem with that. I suggest something like:
```suggestion
                if len(line_list) != 1:
                    previous_event_timediff = line_list[index + 1]["timediff"]
                else:
                    previous_event_timediff = rpevent["timediff"]
```

Reply:
Nice catch! Fixed